easybuild/easyblocks/g/gcc.py

```diff
@@ -387,7 +457,27 @@ def configure_step(self):
                     self.cfg.update('configopts', "--disable-sjlj-exceptions")
                     self.cfg.update('configopts', "--enable-newlib-io-long-long")
                     self.cfg['configure_cmd_prefix'] = '../'
-                return super(EB_GCC, self).configure_step()
+                    return super(EB_GCC, self).configure_step()
+                elif self.build_stages[self.iter_idx] == AMD_NEWLIB:
+                    # compile nvptx target compiler
```

Comment:
same here, lots of extra code, deserves to live in a separate method (so you can use `self.configure_amd_newlib()` here)

Reply:
With the new reductions in code, do you still think moving this to a separate method makes sense? I'm leaning towards not moving the code since it doesn't make sense as a separate method and I think it will only lead to indirection when needing to change the code in the future.